scipy/stats/tests/test_qmc.py

```diff
@@ -490,7 +490,7 @@ def test_MultivariateNormalQMCShapiro(self):
         engine = qmc.MultivariateNormalQMC(
             mean=[0, 0], cov=[[1, 0], [0, 1]], seed=12345
         )
-        samples = engine.random(n_samples=250)
+        samples = engine.random(n_samples=1000)
```

Comment:
here we should definitely do 1024 instead

Reply:
Thanks for the review! Indeed I again forgot to use 2^n points. I will also update the doctests.

BTW, Is the warning enough in Sobol's class docstring?